cluster/aws/options.md

```diff
@@ -88,4 +88,26 @@ Defaults to vivid (Ubuntu Vivid Vervet), which has a modern kernel and does not
 Other options may require reboots, updates or configuration, and should be used only if you have a compelling
 requirement to do so.
 
+**NON_MASQUERADE_CIDR**
+
+The 'internal' IP range which Kuberenetes will use, which will therefore not
+use IP masquerade.  By default kubernetes runs an internal network for traffic
+between pods (and between pods and services), and by default this uses the
+`10.0.0.0/8` range.  However, this sometimes overlaps with a range that you may
+want to use; in particular the range cannot be used with EC2 ClassicLink.  You
+may also want to run kubernetes in an existing VPC where you have chosen a CIDR
+in the `10.0.0.0/8` range.
+
+Setting this flag allows you to change this internal network CIDR.  A sensible
+value is `100.64.0.0/10` - this is also a reserved range, and is much less
```

Comment:
Hmm, it's reserved for a purpose - is it really safe to use?

On Thu, Jan 21, 2016 at 8:49 AM, Justin Santa Barbara <
notifications@github.com> wrote:

> In cluster/aws/options.md
> <https://github.com/kubernetes/kubernetes/pull/19389#discussion_r50427392>
> :
>
> > @@ -88,4 +88,26 @@ Defaults to vivid (Ubuntu Vivid Vervet), which has a modern kernel and does not
> >  Other options may require reboots, updates or configuration, and should be used only if you have a compelling
> >  requirement to do so.
> >
> > +**NON_MASQUERADE_CIDR**
> > +
> > +The 'internal' IP range which Kuberenetes will use, which will therefore not
> > +use IP masquerade.  By default kubernetes runs an internal network for traffic
> > +between pods (and between pods and services), and by default this uses the
> > +`10.0.0.0/8` <http://10.0.0.0/8> range.  However, this sometimes overlaps with a range that you may
> > +want to use; in particular the range cannot be used with EC2 ClassicLink.  You
> > +may also want to run kubernetes in an existing VPC where you have chosen a CIDR
> > +in the `10.0.0.0/8` <http://10.0.0.0/8> range.
> > +
> > +Setting this flag allows you to change this internal network CIDR.  A sensible
> > +value is `100.64.0.0/10` <http://100.64.0.0/10> - this is also a reserved range, and is much less
>
> 100.64 - it's another (not widely used) reserved CIDR ...
> https://en.wikipedia.org/wiki/Reserved_IP_addresses
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/19389/files#r50427392>.
>